src/commands/ticker/index/processor.ts

```diff
@@ -344,15 +379,16 @@ export async function viewTickerInfo(
   interaction: ButtonInteraction,
   msg: Message
 ) {
-  const [coinId, days, symbol, discordId] = interaction.customId
+  const [coinId, days, symbol, isDChart, discordId] = interaction.customId
     .split("|")
     .slice(1)
   const { messageOptions } = await composeTokenInfoEmbed(
     msg,
     coinId,
     +days,
     symbol,
-    discordId
+    discordId,
+    JSON.parse(isDChart)
```

Comment:
`JSON.parse` sẽ throw exception nếu input pass vào là invalid string
```suggestion
    isDominanceChart: String(isDChart).toLowerCase() === "true",
```